### PR TITLE
Extend curve building to 100 years

### PR DESCRIPTION
**Pull request by tleitch**
Fork of master, modified, tested with R CMD check and travis

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,5 +1,11 @@
 2016-05-12  Dirk Eddelbuettel  <edd@debian.org>
```
Comment (eddelbuettel) [inline ChangeLog]: You want today's date and your name here, I suppose.

**Comment by eddelbuettel** [comment]
Looks good -- three commits, four files, clean diff that can be merged.  One minor nag noted, you should be able to alter and push to this branch extending the PR to four commits.

We may be getting the hang of it :)

**Comment by tleitch** [comment]
Thanks.Note: two commits were artificially generated to trigger travis build for the first time. Are you suggesting there should be 1 commit per file?

**Comment by eddelbuettel** [comment]
No, I was just counting.   Yes, each commit (and push) triggers a build (unless you add `[ci skip]` to the subject).    

Tastes vary on whether many commits, or aggregated commits into one (ie "squashed") are better.  Larger projects require squashed.    A "handful" just like here is fine by me.

**Comment by eddelbuettel** [comment]
It's merged.  If you declare my repo as an additional remote (the term 'parentrepo' is often used) then `git pull --all` will get updates from both, and can merge my changes (ie from your PR plus my merge commit) and push that back to your fork so that both are in sync.

**Pull request merged**